### PR TITLE
Schema supports use of `<to>`, `<data>`, `<value>` and multi-sign XML

### PR DESCRIPTION
**Pull request by colourful-land**

### CODE REVIEW

File: schema/tokenscript.xsd [hunk 1]
```diff
@@ -297,14 +297,14 @@
     </xs:element>
     <xs:element name="ethereum">
         <xs:complexType>
-            <xs:choice maxOccurs="2">
+            <xs:sequence>
+                <xs:element minOccurs="0" name="to" type="address"/>
```
Comment (James-Sangalli) [inline schema/tokenscript.xsd]: should still have minOccurs of 1? Cannot send a transaction to nobody

Comment (colourful-land) [reply schema/tokenscript.xsd]: You can have an empty `<ethereum>` tag in 2019-05 schema (although that might change in the future, but you can now.)

It happens when you want to call a function that has no parameters nor values needed.

Comment (James-Sangalli) [reply schema/tokenscript.xsd]: @colourful-land this is assuming that the 'to' param is referenced by contract name, correct?

Comment (colourful-land) [reply schema/tokenscript.xsd]: No. That is what I proposed in #191. This PR is about what is already decided on the Monday Meeting. We need to talk over the phone. It's useless to write to clarify writing.

Comment (colourful-land) [reply schema/tokenscript.xsd]: This has absolutely nothing to do with #191. This is needed even we decided #191 is not the way to go

**Pull request merged**